spaces/domains.json

```
@@ -292,7 +292,9 @@
   "vote.to-the-moon.farm": "to-the-moon-farm.eth",
   "vote.polyvertex.finance": "polyvertex.eth",
   "gov.aol.gg": "apesonline.eth",
+  "gov.forefront.market": "ffdao.eth",
   "gov.coin-bid.io": "coin-bid.eth",
   "vote.sil.finance": "sister-in-law.eth",
+  "vote.forefront.market": "ffdao.eth",
```

You will need to choose only one subdomain. Please delete one of them.
```suggestion
  "gov.forefront.market": "ffdao.eth",
  "gov.coin-bid.io": "coin-bid.eth",
  "vote.sil.finance": "sister-in-law.eth",
```